ocaml/graph/README.markdown

```diff
@@ -10,24 +9,27 @@ To create a graph showing relationships between
 
 run:
 
-  $ graph -h <host> -u root -pw <password> VM VDI VBD SR > graph.gv
+```bash
+graph.native -h <host> -u root -pw <password> VM VDI VBD SR > graph.dot
+```
 
-install graphviz and then run:
+Then install graphviz and run:
 
-  $ dot -Tpng graph.gv -o graph.png
+```bash
+dot -Tsvg graph.dot -o graph.svg
+```
 
 Here's an example:
 
-![Example rendered graph](../../../../raw/master/ocaml/graph/example.png)
+![Example rendered graph](vm-disks.svg)
```

Comment:
On GitHub, this will just be a link to the svg for some reason: https://github.com/kc284/xen-api/tree/master/ocaml/graph . But that's absolutely fine, just wanted to mention it.